packages/ppx_compose/ppx_compose.0.0.3/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "ppx_compose"
+authors: ["Petter A. Urkedal"]
+maintainer: "paurkedal@gmail.com"
+homepage: "https://github.com/paurkedal/ppx_compose/"
+bug-reports: "https://github.com/paurkedal/ppx_compose/issues"
+dev-repo: "https://github.com/paurkedal/ppx_compose.git"
+license: "LGPL-3 with OCaml linking exception"
+
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+
+depends: [
+  "ocamlbuild" {build}
```

Comment:
Indeed, I'll make it `{build & >= "1.0+beta7"}` even.  Unless you think otherwise, I'll skip the corresponding fix in the tarball till the next release.